doc/nrf/ug_nrf52.rst

```diff
@@ -45,6 +45,13 @@ Refer to the PCA number for building projects for these boards and find a link t
      - PCA10040
      - :ref:`nrf52810_pca10040<zephyr:nrf52810_pca10040>`
 
+nRF Desktop
+===========
+
+As reference for a complete project that uses Bluetooth Low Energy, see the :ref:`nrf_desktop` application.
```

Comment:
```suggestion
The nRF Desktop application is a complete project that integrates Bluetooth Low Energy, see the :ref:`nrf_desktop` application.  It can be built for the nRF Desktop reference hardware or an nRF52840 DK.
```